src/MagnumPlugins/UfbxImporter/UfbxImporter.cpp

```diff
@@ -0,0 +1,1052 @@
+
+        attributeData[attributeIndex++] = MeshAttributeData{
+            MeshAttribute::Normal, normals};
+        attributeOffset += sizeof(Vector3);
+    }
+
+    if (mesh->vertex_tangent.exists) {
+        Containers::StridedArrayView1D<Vector3> tangents{vertexData,
+            reinterpret_cast<Vector3*>(vertexData + attributeOffset),
+            vertexCount, stride};
+
+        attributeData[attributeIndex++] = MeshAttributeData{
+            MeshAttribute::Tangent, tangents};
+        attributeOffset += sizeof(Vector3);
+    }
+
+    if (mesh->vertex_bitangent.exists) {
+        Containers::StridedArrayView1D<Vector3> bitangents{vertexData,
+            reinterpret_cast<Vector3*>(vertexData + attributeOffset),
+            vertexCount, stride};
+
+        attributeData[attributeIndex++] = MeshAttributeData{
+            MeshAttribute::Bitangent, bitangents};
+        attributeOffset += sizeof(Vector3);
+    }
+
+    for (std::size_t set = 0; set < mesh->uv_sets.count; set++) {
+        Containers::StridedArrayView1D<Vector2> bitangents{vertexData,
+            reinterpret_cast<Vector2*>(vertexData + attributeOffset),
+            vertexCount, stride};
+
+        attributeData[attributeIndex++] = MeshAttributeData{
+            MeshAttribute::TextureCoordinates, bitangents};
+        attributeOffset += sizeof(Vector2);
+    }
+
+    for (std::size_t set = 0; set < mesh->color_sets.count; set++) {
+        Containers::StridedArrayView1D<Color4> bitangents{vertexData,
+            reinterpret_cast<Color4*>(vertexData + attributeOffset),
+            vertexCount, stride};
+
+        attributeData[attributeIndex++] = MeshAttributeData{
+            MeshAttribute::Color, bitangents};
+        attributeOffset += sizeof(Color4);
+    }
+
+    CORRADE_INTERNAL_ASSERT(attributeIndex == attributeCount);
+    CORRADE_INTERNAL_ASSERT(attributeOffset == stride);
+
+    return MeshData{MeshPrimitive::Triangles,
+        std::move(indexData), MeshIndexData{indices},
+        std::move(vertexData), std::move(attributeData),
+        (UnsignedInt)vertexCount, mesh};
+}
+
+UnsignedInt UfbxImporter::doMaterialCount() const {
+    return (UnsignedInt)_state->scene->materials.count;
+}
+
+Int UfbxImporter::doMaterialForName(Containers::StringView name) {
+    return typedId(ufbx_find_element_len(_state->scene.get(), UFBX_ELEMENT_MATERIAL, name.data(), name.size()));
+}
+
+Containers::String UfbxImporter::doMaterialName(UnsignedInt id) {
+    return _state->scene->materials[id]->name;
+}
+
+Containers::Optional<MaterialData> UfbxImporter::doMaterial(UnsignedInt id) {
+    ufbx_material *material = _state->scene->materials[id];
+
+    struct MaterialMappingList {
+        Containers::ArrayView<const MaterialMapping> mappings;
+        Containers::ArrayView<ufbx_material_map> maps;
+    };
+
+    MaterialMappingList mappingLists[] = {
+        { materialMapsFbx, material->fbx.maps },
+        { materialMapsPbr, material->pbr.maps },
```

Comment:
To fix the cursed CI error on the ancient GCC 4.8, this might hopefully help?

```suggestion
        {Containers::arrayView(materialMapsFbx), Containers::arrayView(material->fbx.maps)},
        {Containers::arrayView(materialMapsPbr), Containers::arrayView(material->pbr.maps)},
```